src/endpoints/AliasesEndpoint.js

```diff
@@ -20,28 +20,28 @@ import Alias from "../models/Alias.js";
 	async getAliasesForAccount() {
 		let aliasesArray = await this.request("aliases");
 
-		return aliasesArray.map(aliasDTO => new Alias(aliasDTO));
+		return aliasesArray.map(aliasDTO => new Alias({...aliasDTO, data: aliasDTO.payload}));
 	}
 
 	/**
 	 * Create a new alias associated with the currently logged in account.
 	 *
 	 * Raises an error if the alias already exists.
 	 * @param {string} name the name of the alias to create
-	 * @param {string } payload payload to attach to the new alias
+	 * @param {Object} data payload to attach to the new alias
 	 * @returns {Promise<Alias>}
 	 */
-	async createAlias(name, payload) {
+	async createAlias(name, data) {
 		let newAliasDTO = await this.request("aliases", {
 				method: "POST",
 				body: {
 					name,
-					payload,
+					payload: JSON.stringify(data),
```

Comment:
Why the double serialization? Possibly a server issue, @farnazj ?

Reply:
@LeaVerou we're trying to minimize server-side changes for now. the server expects payload to be a string so we stringify payload to a string. request will also serialize the body later but if we remove this stringify call then on the server when we call deserialize payload would be an object instead of a string.